src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -599,10 +601,12 @@ class NetworkOPsImp final : public NetworkOPs
 
     Json::Value
     transJson(
-        const STTx& transaction,
+        std::shared_ptr<STTx const> const& transaction,
         TER result,
+        std::shared_ptr<ReadView const> const& ledger,
         bool validated,
-        std::shared_ptr<ReadView const> const& ledger);
+        std::optional<std::reference_wrapper<TxMeta const>> meta,
+        int apiVersion);
```

Comment:
Since `apiVersion` will always be a positive integer and` Context::apiVersion` is an `unsigned int` [[source]](https://github.com/XRPLF/rippled/blob/3e08c390f5f18b308ace0dbdbb27fa02e49141fa/src/ripple/rpc/Context.h#L50)
Isn't it better to make `apiVersion` in this PR an `unsigned int` as well or am I missing something ?